src/core/Value.cpp

```diff
@@ -277,7 +274,24 @@ std::size_t Value::getIndexInStore( const std::size_t& ival ) const {
   return ival;
 }
 
-double Value::get(const std::size_t& ival, const bool trueind) const {
+bool Value::checkValueIsActiveForMMul(const std::size_t task) const {
+  const auto ncol = getRowLength(task);
+  const auto base = task * getNumberOfColumns();
+  if (hasDeriv) {
+    for(unsigned k=base; k<base+ncol; ++k) {
+      if(std::fabs(data[k*(1+ngrid_der)])>0.0) {
+        return true;
+      }
+    }
+    return false;
+  }
+  return std::any_of(&data[base],&data[base+ncol],[](double x) {
```

Comment:
The funny thing is the AI complains about the commented out code, but not at the very nasty bug that I proudly produced :wink:

```suggestion
  return std::any_of(&data[base],&data[base]+ncol,[](double x) {
```

@gtribello if you accept this suggestion trough the GH UI we solve the bug that the debug runs are catching

Fun Fact" compiling plumed with -g and running the tests cost you ~50 GB of space